apps/cyberstorm-nextjs/app/error.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+
+import { useEffect } from "react";
+import { ErrorLayout } from "@thunderstore/cyberstorm";
+
+export default function Error({ error }: { error: Error; reset: () => void }) {
+  useEffect(() => {
+    console.error(error);
+  }, [error]);
+
+  return <ErrorLayout error={500} />;
+}
```

Comment:
This component should not have `"use client"` nor should it use state for anything.